### ISSUE TITLE
SectionList's sticky headers lose their stickiness when placed within a ScrollView (or another SectionList)

### ISSUE THREAD

**Issue by palexs** [facebook/react-native#15954]
In our app we need to achieve double sticky headers. So there should be 1 common sticky header for entire table and sticky headers for each `SectionList` section.
But whenever I wrap `SectionList` inside container `ScrollView` (or other `SectionList`) its sections headers loose their stickiness. Is there a way to achieve the desired result?

![double-sticky-headers](https://user-images.githubusercontent.com/1583824/30422315-3e00bbfe-9948-11e7-87ce-b070d1f61f86.gif)

So, basically I somehow need to make those "Section Title #" stick under that red "Column Sticky Header". Current implementation (https://snack.expo.io/HJCWpaP9W) wraps `SectionList` inside parent `SectionsList`.

**Comment by radko93** [comment]
@facebook-github-bot stack-overflow
